compose/ts-translation-service.yml

```diff
@@ -0,0 +1,37 @@
+version: '2.4'
+
+services:
+  ts-translation-service:
+    build:
+      context: .
+      args:
+        - http_proxy
+        - https_proxy
+        - no_proxy
+    image: "hmctspublic.azurecr.io/ts/translation-service:latest"
```

Comment:
Can you restructure to match the image values in backend.yml, i.e. using TS specific variables for `_USE_LOCAL` & `_TAG` variable

Reply:
done